docs/ruleset_functions/zone_meets_G3_1_1e.md

```diff
@@ -0,0 +1,60 @@
+- set the result variable to "No" - only a positive test can give it a different value: `result = "No"`
+- set eligibility boolean to False: `eligible = TRUE`
+- For each space in zone: `for space_p in zone.Spaces:`
+	- check if the space has an eligible lighting type: `if space_p.lighting_space_type not in eligible_space_types:`
+		- set eligibility to False - any non-compliant space type will result in a non-eligible zone: `eligible = FALSE`
+
+- if the zone is not eligible, check if it is a vestibule: `if eligible == FALSE:`
+	- set variable is_a_vestibule = is_zone_a_vestibule: `is_a_vestibule = is_zone_a_vestibule(zone, B-RMR)`
+	- if the result is not equal to no, then it is eligible to be a vestibule: `if is_a_vestibule != "NO":`
+		- set eligible to true: `eligible = TRUE`
+
+- if the zone is still eligible: `if eligible == TRUE:`
+	- check if the proposed has a heating-only system
+	- get the building_segment_p the zone is in:
+	`for building_segment_p in P-RMR.building.building_segments:`
+		- check if the zone is in the building_segment:
+		`if zone in building_segment.zones:`
+			- break - this means the variable "building_segment" is the building_segment that contains the zone:
+			`break`
+	- loop through each heating system: `for system_p in building_segment.heating_ventilation_air_conditioning_systems:`
+		- look for a cooling system: `if system_p.cooling_system != "Null":`
+			- check to see if the cooling_system_options equals "None": `if system_p.cooling_system.cooling_system_options == "None":`
+				- the zone is eligible
+			- otherwise: `else:`
+				- there is a cooling system and the zone is not eligible: `eligible = FALSE`
+
+- if the zone is still eligible: `if eligible == TRUE:`
+	- set result to "E": `result = "E"`
+	- however, if there is transfer air into the space from an air conditioned zone, it is not eligible, so look for transfer air into the space: `if zone.transfer_airflow_rate > 0:`
+		- there is no way of knowing where the transfer air is coming from, so for now will give the result the tag "E_T" to signal that there is transfer air.  When the rule is written, "E_T" from this function will trigger an "UNDETERMINED" result and indicate manual checking for transfer air from air-conditioned space: `result = "E_T"
+
+
+**Returns** `result`
+
+
+**Notes/Questions:**  
+1. the "is_zone_a_vestibule" function gives only "NO" and indeterminite answers.  Is there a way to alert to confirm that the zone is a vestibule?
+2. Transfer air does not indicate whether it is transferring from an air conditioned zone or not, therefore cannot do a complete check, only an alert
+	221006 - Jason suggests that the schema gets modified to indicate where the transfer air is coming from ("transfer_air_from_zone")
+3. does a HeatingVentilationAirConditioningSystem always have a CoolingSystem, or does it return "Null" if there is no cooling system?

```

Comment:
`cooling_system` is not a required data in the schema, so it can be empty.
In python, getting a non-existent `cooling system` will return `None`. 

When dealing with non-required data, we can decide whether to stop the process and return `UNDETERMINED` decision or accept `None` as one possible outcome.